### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by jondricek** [ralphje/imagemounter#39]
I see that this repository hasn't had many updates in the past couple years. it is still maintained? I help maintain a fork of this over at https://github.com/mitre/imagemounter (which itself is a bit behind the master branch here) and am wondering how long it would take to get updates pulled into this upstream repository and eventually onto PyPI as well.

Alternately, I am contemplating publishing our fork (after it is cleaned up a bit) to PyPI as well as something like `imagemounter2` but don't want to jump right into that if it makes sense to collaborate more in tandem with you. Thanks!

**Comment by ralphje** [comment]
Hi,

Thanks for reaching out. I'm still planning to actively develop on this, but haven't found the time... 'recently'. I'll be happy to work together with you on updating this repository.

In the coming months, I might have more time and a refreshed engagement would also be beneficial for me getting around to work on it.

Sorry for letting this repository sitting a bit dormant.

**Comment by jondricek** [comment]
No worries at all! The fork I've got is something like 1 commit ahead/40 behind yours anyway, so not exactly up to date as it is. I just got to a point on my project though that it would be beneficial to pip install some of the changes we've done on our fork without requiring git to be installed since that is how we currently have it set up and it became a discussion point of "well, what do we want to do exactly then?"

Thanks for maintaining this repo - it has been immensely helpful over the years. I hope to get a pull request set up in the next week or two and try to jog my own memory as to what our own single commit was even about.
